apps/menu-bar/modules/menu-bar/ios/MenuBarModule.swift

```diff
@@ -0,0 +1,221 @@
+import ExpoModulesCore
+import ServiceManagement
+
+let onNewCommandLine = "onNewCommandLine"
+let onCLIOutput = "onCLIOutput"
```

Comment:
Let's use more restrictive access levels, especially for globals.
```suggestion
private let onNewCommandLine = "onNewCommandLine"
private let onCLIOutput = "onCLIOutput"
```